sdk/healthdataaiservices/Azure.ResourceManager.HealthDataAIServices/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (2024-05-11)
```

Comment:
is this a typo? Now we are in November but here is a May

Reply:
Shame on me, that is a typo I was aiming for 2024-11-11